apps/subs_web/frontend/webpack.config.js

```diff
@@ -47,6 +49,19 @@ module.exports = {
     path: path.resolve(__dirname, '../priv/static'),
     filename: '[name].bundle.js',
   },
+  /*
+  optimization: {
```

Comment:
Can we remove this commented code?

Reply:
I added another commit to the pull request to remove the front end changes.